### Delays Analysys
*In this project we are going to read the information from our .CSV file into pandas,and perform analysis to compare the arrival delays for the two airlines*

In [1]:
# bring in NumPy and pandas
import numpy as np
import pandas as pd

df = pd.read_csv('./csv/delays9-16-17.csv')
df

id  airline    destination  on_time  delayed
0   1   ALASKA    Los Angeles      497       62
1   2   ALASKA        Phoenix      221       12
2   3   ALASKA      San Diego      212       20
3   4   ALASKA  San Francisco      503      102
4   5   ALASKA        Seattle     1841      305
5   6  AM WEST    Los Angeles      694      117
6   7  AM WEST        Phoenix     4840      415
7   8  AM WEST      San Diego      383       65
8   9  AM WEST  San Francisco      320      129
9  10  AM WEST        Seattle      201       61

#### Here we Create variables that hold data for $ALASKA$ and $AM WEST$ airlines

In [57]:
alaska = df.loc[df['airline'] == 'ALASKA']
amWest = df.loc[df['airline'] == 'AM WEST']

#### Now we'll find out the total delays for each airline:

In [58]:
alaskaDelayed = sum(alaska['delayed'])
amWestDelayed = sum(amWest['delayed'])
print("ALASKA airline was dealyed %s times and AM WEST %s times." % (alaskaDelayed, amWestDelayed))
print("Whick makes a difference of %s delays." % abs(alaskaDelayed - amWestDelayed))

ALASKA airline was dealyed 501 times and AM WEST 787 times.
Whick makes a difference of 286 delays.


#### Destination with top delays for $ALASKA$ airline is Seatle, that counts 305 delays.

In [59]:
topAlasksDelay = alaska.loc[alaska['delayed'].idxmax()]
topAlasksDelay['destination'] , topAlasksDelay['delayed']

('Seattle', 305)

#### Destination with top delays for $AM WEST$ airline is Phoenix, that counts 415 delays.

In [60]:
topAmWestDelay = amWest.loc[amWest['delayed'].idxmax()]
topAmWestDelay['destination'] , topAmWestDelay['delayed']

('Phoenix', 415)

### Conclusion
After analysing this data, we determined that AM WEST airlines have 286 delays more than ALASKA airline.
We also found out that the most delays for ALASKA airlines happen in Seattle, and in case of AM WEST airline the most delays happen in Phoenix.